format/Message.fbs

```diff
@@ -0,0 +1,113 @@
+namespace arrow.flatbuf;
```

Comment:
sorry if I missed it on the e-mail list but is there a spec writeup for this (I seem to recall that maybe that was a follow-up JIRA item for this).